internal/resources/slo/resource_slo.go

```diff
@@ -298,19 +298,42 @@ func packSloResource(d *schema.ResourceData) (gapi.Slo, error) {
 		tflabels = packLabels(labels)
 	}
 
-	alerting := d.Get("alerting").([]interface{})
-	if len(alerting) > 0 {
-		alert := alerting[0].(map[string]interface{})
-		tfalerting = packAlerting(alert)
+	alerting, alertingOK := d.GetOk("alerting")
```

Comment:
It's more idiomatic to collapse this into the conditional - you don't need to worry about alerting, and alertingOK "contaminating" the scope outside of the block in that case:
```go
if alerting, ok := d.GetOk("alerting"); ok {
  ...
}

// alerting is no longer declared here, compile-time error:
alerting = nil
```

Reply:
Agreed with ☝🏽